sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobAsyncClient.java

```diff
@@ -359,6 +368,27 @@ as we can guarantee we only need at most two buffers for any call to write (two
         }
     }
 
+    private Mono<Response<BlockBlobItem>> determineUploadFullOrChunked(final Flux<ByteBuffer> data,
+        final Function<Flux<ByteBuffer>, Mono<Response<BlockBlobItem>>> path1,
```

Comment:
nit: naming `path1` -> `stageAndUpload`, `path2` -> `upload` or something similar

Reply:
Changed to `uploadInChunks` and `uploadFullBlob`